frontend/src/views/CanteenEditor.vue

```diff
@@ -332,6 +366,21 @@ export default {
         delete e["returnValue"]
       }
     },
+    deleteCanteen() {
+      this.$store
+        .dispatch("deleteCanteen", { id: this.canteen.id })
+        .then(() => {
+          this.bypassLeaveWarning = true
+          this.$store.dispatch("notify", {
+            message: "Votre cantine a bien été supprimée",
```

Comment:
```suggestion
            message: "Votre cantine a bien été supprimée. Si c'était par accident, contactez nous sur contact@egalim.beta.gouv.fr.",
```
Ou qqch pour éviter du panique

Reply:
Ajouté sur https://github.com/betagouv/ma-cantine/pull/323/commits/be71c908b9690a291ef168e8e6fb05a31c75cdf2